app/code/Magento/CheckoutAgreements/Block/Adminhtml/Agreement/Edit/Form.php

```diff
@@ -16,13 +16,31 @@ class Form extends \Magento\Backend\Block\Widget\Form\Generic
      * @var \Magento\CheckoutAgreements\Model\AgreementModeOptions
      */
     protected $agreementModeOptions;
+    
+    /**
+     * @var \Magento\Cms\Model\Wysiwyg\Config
+     */
+    protected $_wysiwygConfig;
```

Comment:
Please change the property access level to `private` and remove the `_` prefix from the property name (as recommended by PSR-2 https://www.php-fig.org/psr/psr-2/#42-properties)